docs/sql/statements/create-table.rst

```diff
@@ -417,6 +417,30 @@ Allows to have a read only table that additionally can be deleted.
   Disables the table settings modifications if set to ``true``, if set
   to ``false`` — table settings modifications are enabled.
 
+.. _table_parameter.soft_deletes.enabled:
+
+``soft_deletes.enabled``
+------------------------
+
+Indicates whether soft deletes are enabled or disabled. 
+
+Soft deletes allow CrateDB to preserve recent deletions within the Lucene
+index. This information is used for shard recovery.
+
+Before the introduction of soft deletes, CrateDB had to retain the information
+in the Translog. Using soft deletes uses less storage than the Translog
```

Comment:
might be beneficial to link "Translog" to https://crate.io/docs/crate/reference/en/4.2/concepts/storage-consistency.html#durability